build/lib/release.sh

```diff
@@ -383,12 +383,15 @@ EOF
           echo "COPY nsswitch.conf /etc/" >> "${docker_file_path}"
         fi
 
-        DOCKER_CLI_EXPERIMENTAL=enabled "${DOCKER[@]}" buildx build \
+        local build_log="${docker_build_path}/build.log"
+        if ! DOCKER_CLI_EXPERIMENTAL=enabled "${DOCKER[@]}" buildx build \
           --platform linux/"${arch}" \
           --load ${docker_build_opts:+"${docker_build_opts}"} \
-          -q \
           -t "${docker_image_tag}" \
-          "${docker_build_path}" >/dev/null
+          "${docker_build_path}" >"${build_log}" 2>&1; then
```

Comment:
LGTM

do we want to cleanup the file?

Reply:
SGTM, @saschagrunert want to add that?

Reply:
Yes, added a `rm "${build_log}"` after the `if` condition. :+1: